pythonforandroid/recipes/mysql-connector-python/__init__.py

```diff
@@ -0,0 +1,17 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+
+
+class MySQLConnectorPythonRecipe(CompiledComponentsPythonRecipe):
+
+    name = 'mysql-connector-python'
+    version = '8.3.0'
+    url = (
+        f"https://downloads.mysql.com/archives/get/p/29/file/"
+        f"mysql-connector-python-py{version}-1ubuntu23.10_amd64.deb"
+    )
```

Comment:
This is giving a 404
```
wget https://downloads.mysql.com/archives/get/p/29/file/mysql-connector-python-py8.3.0-1ubuntu23.10_amd64.deb
```

Why didn't you keep it as :
```
https://dev.mysql.com/get/Downloads/Connector-Python/mysql-connector-python-{version}-src.tar.gz
```
Or with line split:
```
    url = (
        "https://dev.mysql.com/get/Downloads/Connector-Python/"
        f"mysql-connector-python-{version}-src.tar.gz"
    )
```

Reply:
i change the link to url = (
        "https://dev.mysql.com/get/Downloads/Connector-Python/"
        f"mysql-connector-python-{version}-src.tar.gz"
    ). Thank you! i home this time will work!